types/Request.d.ts

```diff
@@ -0,0 +1,22 @@
+declare type OriginReq = {
+  method: string,
+  answer: Function,
+  source?: string,
+}
+
+declare type RequestEventPayload = {
+  [key: string]: any,
+}
+
+declare type RequestEventHandler = (payload: RequestEventPayload, req: OriginReq) => void;
+declare type RequestEventHandlers = {
+  [key: string]: RequestEventHandler,
+}
+
+declare type ConnectErrors = import('ConnectError').ERRORS;
```

Comment:
please, define other name for errors, because `ConnectErrors` and `ConnectError` is like the same, but they are not
maybe `CONNECT_ERRORS` will be better? because this is constants

Reply:
Type isn't a constant. So we can't mix constant naming convention with types naming convention - it's a bad practice. I'll try to find more readable name for that.

Reply:
Unnecessary type.